src/runner.ts

```diff
@@ -74,18 +74,18 @@ export class StaticAnalysisParserRunner {
         await this.uploadReportResultsToBitbucket();
     }
 
-    private getBitbucketEnvs(): BitbucketEnv {
-        const requiredEnvs: BitbucketEnv = {
-            USER_EMAIL: process.env.USER_EMAIL || '',
-            API_TOKEN: process.env.API_TOKEN || '',
+    private getBitbucketEnvs(): BitbucketEnvs {
+        const requiredEnvs: BitbucketEnvs = {
+            USERNAME: process.env.USERNAME || '',
```

Comment:
I think this should be consistent with what has been used in configuration. For example here it should be the username, but I tried using my email earlier, and it didn’t work. The variable name confused me. The same goes for the app password.